yogstation/code/modules/antagonists/darkspawn/psi_web.dm

```diff
@@ -0,0 +1,141 @@
+//shadow store datums (upgrades and abilities)
```

Comment:
You can put an underscore at the start of this file name to ensure it shows up first before the rest and place it in the darkspawn upgrades folder, since it's the base of the others